front-end/src/types/index.ts

```diff
@@ -0,0 +1,12 @@
+import { Key } from "react";
+
+interface TodoType {
+    createdAt: string;
```

Comment:
Any specific reason to keep `createdAt` as `string` in place of JS `Date` type?

Reply:
That is actually easier. usko kabhibhiDAte mei convert kr skte hai, aur string type mei errors nhi aate. new Date() date return krta hai jo error produce krta hai in hooks, Date() string return krta hai, that is way easier to handle around.

Reply:
related types can be done in one file and then imported into index.ts